Bucketeer.xcodeproj/project.pbxproj

```diff
@@ -1054,6 +1055,27 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		9BC3E3082AE91D7700849EB8 /* Run Script(linter) */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Run Script(linter)";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "#Workaround: Add Homebrew Path for M1 Mac.\nif [ $(uname -m) = \"arm64\" ]; then\n  export PATH=\"/opt/homebrew/bin:/opt/homebrew/sbin:${PATH+:$PATH}\";\nfi\n\nif which mint >/dev/null; then\n  make run-lint\nelse\n  echo \"Warning: Mint is installed. Please run make install-mint.\"\nfi\n";
```

Comment:
Hi @kakcy , I'd like to add a small improvement.

We could check to see if we are in the CI environment or not. `env CI=true | false`
This postscript only runs if the env `CI` is `false` or `undefined` 

So that we will not depend on the `lint-install` for the `build`, and `test` stages

while keep it continuing to work on the  local machine

Reply:
what do you think ?

Reply:
Hi @duyhungtnn, Thank you for your suggestions for improvement.

I also think it's better that way, so I'll try to fix it.